java/messages/src/main/java/com/nedap/retail/messages/epcis/v1_1/EpcisEventContainer.java

```diff
@@ -17,12 +18,11 @@
     public EpcisEventContainer() {
     }
 
-    public EpcisEventContainer(final List<EpcisEvent> events) {
-        this.events = events;
+    public EpcisEventContainer(final List<? extends EpcisEvent> events) {
+        this.events = new ArrayList<>(events);
```

Comment:
Vague comment: _"Added more flexibility to constructor"_.
What is the reason behind this change?